{
    "type": "bundle",
    "id": "bundle--38526ffa-9a2f-41e5-8094-271f2415c55e",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--cce60b2e-094d-4792-a677-8ca5acb52886",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-07-02T01:00:54.584183Z",
            "modified": "2024-07-02T01:00:54.584183Z",
            "name": "CVE-2024-39427",
            "description": "In trusty service, there is a possible out of bounds write due to a missing bounds check. This could lead to local denial of service with System execution privileges needed",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-39427"
                }
            ]
        }
    ]
}